core/test/time.jl

```diff
@@ -0,0 +1,13 @@
+using Ribasim
+
+@testset "Time dependent flow boundary" begin
+    toml_path = normpath(@__DIR__, "../../data/flow_boundary_time/flow_boundary_time.toml")
+    @test ispath(toml_path)
+    model = Ribasim.run(toml_path)
+
+    t = model.saved_flow.t
+    flow = [flows[1] for flows in model.saved_flow.saveval]
+    flow_expected = 1 .+ sin.(0.5 * π * model.saved_flow.t ./ t[end]) .^ 2
```

Comment:
It's fine either way, but FYI if you get tired of the dots, you can use the `@.` macro: `@. y = x + 3 * sin(x)` does everything elementwise.

Reply:
It seems that your usage of the macro is not quite right, it should be `y = @. x + 3 * sin(x)`.

Reply:
`@. y = x + 3 * sin(x)` also works but only if you already have a vector y, then it fills it in place, if you don't, then indeed your version is the right one to use.